public/components/custom_panels/panel_modules/visualization_container/visualization_container.tsx

```diff
@@ -212,7 +212,7 @@ export const VisualizationContainer = ({
   ];
 
   if (usedInNotebooks) {
-    popoverPanel = catalogVisualization ? [showModelPanel] : [popoverPanel[0]];
+    popoverPanel = catalogVisualization ? [showModelPanel] : [popoverPanel[0], showModelPanel];
```

Comment:
`showModelPanel` Shouldn't be added here. Any reason you added this here?

Reply:
user should be able to view the query of any particular metric without being navigated back to event analytics just for viewing it. Previously this is how it worked, idk how it got changed later

Reply:
Removed it now, but this should be discussed with Kyle to change it uniformly across all plugins